frontend/src/components/Dashboard.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import styled from "styled-components/macro";
+import Header from "../commons/Header";
+import Buttons from "../commons/Buttons";
+import {useHistory} from "react-router-dom";
+import usePositioning from "../hooks/usePositioning";
+
+export default function Dashboard() {
+
+    const history = useHistory();
+
+    const [positionAvailable, latitude, longitude, errorMessage] = usePositioning();
+
+    return (
+        <PageLayout>
+            <Header headerText={"Ahoi 'username'"}/>
+            <PositionBlock>
+                <div>Current location: {positionAvailable ? <b>Available</b> : <b>Not Available</b>}</div>
+                {positionAvailable ?
+                    <div>
+                        <div>Latitude: <b>{latitude}</b></div>
+                        <div>Longitude: <b>{longitude}</b></div>
+                    </div>
+                    :
+                    <div id={"error"}>{errorMessage}</div>
+                }
+            </PositionBlock>
+            <div>some content</div>
+            <Buttons
+                disableButtonOne={false}
+                labelButtonOne={"Log Out"}
+                disableButtonThree={false}
+                labelButtonThree={"Input"}
+                clickHandlerThree={clickHandlerInput}
+            />
+        </PageLayout>
+    );
+
+    function clickHandlerInput() {
```

Comment:
Try a verb oriented naimng style for functions, better say what it does, not what it is.